tests/postconditions/pass/assignment_poststate.act

```diff
@@ -0,0 +1,14 @@
+constructor of C
```

Comment:
This test is added in the postcodnitions directory, so it should have an `ensures` block.